apps/docs/content/docs/introduction.mdx

```diff
@@ -8,26 +8,22 @@ order: 0
 
 Welcome to the tldraw developer docs.
 
-Here at tldraw, we make two things: a very good multiplayer whiteboard (at [tldraw.com](https://www.tldraw.com)) and the [open source](https://github.com/tldraw/tldraw) libraries used to create that product. This page is meant to provide documentation and reference for those open source libraries.
+Here at tldraw, we make two things: a very good multiplayer whiteboard (at [tldraw.com](https://tldraw.com)), and the [open source library](https://github.com/tldraw/tldraw) that powers it. This page provides documentation and reference for that open source library.
 
 ```tsx 
 import { Tldraw } from '@tldraw/tldraw'
 import '@tldraw/tldraw/tldraw.css'
 
 export default function () {
-	return (
-		<div style={{ position: 'fixed', inset: 0 }}>
-			<Tldraw />
-		</div>
-	)
+	return <Tldraw />
```

Comment:
Maybe radical (and tricky) but I would love for us to be able to say: "Just return `<Tldraw />`"